src/main/java/codesquad/springcafe/domain/article/Article.java

```diff
@@ -0,0 +1,78 @@
+package codesquad.springcafe.domain.article;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class Article {
+    private long id;
```

Comment:
```suggestion
    private Long id;
```
1% 정도 Long 클래스를 선호하는 느낌이 있네요.

Reply:
DB를 사용할 땐 `auto_increment` 를 사용하기 때문에 not null 이 보장됩니다.
그래서 원시 타입을 사용했는데, 생각해보니 현재 단계에선 Repository의 저장하는 자료구조가 `Map<Long, Member>`라서 오토박싱하는 문제가 있네요..